BlueprintUICommonControls/Sources/Label.swift

```diff
@@ -6,31 +6,78 @@ import UIKit
 public struct Label: ProxyElement {
```

Comment:
The changes in here make it so we cache the allocation of the attributed string, attributed label, etc, which matters for deep hierarchies which re-measure a lot